From a92653a6e313c93c7a624407ab770a6b89f729bd Mon Sep 17 00:00:00 2001
From: Jan Hannig <jan.hannig@iris-sensing.com>
Date: Mon, 28 Aug 2023 15:27:31 +0200
Subject: [PATCH] epc660: remove the register setting of EPC660_REG_LED_DRIVER
 0x90

as the register should be only settable by platform application.

Signed-off-by: Jan Hannig <jan.hannig@iris-sensing.com>
Signed-off-by: Jasper Orschulko <jasper@fancydomain.eu>
---
 drivers/media/i2c/epc660.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/epc660.c b/drivers/media/i2c/epc660.c
index 75a4b5f145ed..3f0449c5ea08 100644
--- a/drivers/media/i2c/epc660.c
+++ b/drivers/media/i2c/epc660.c
@@ -202,8 +202,9 @@ static int epc660_s_stream(struct v4l2_subdev *sd, int enable)
 		/* Enable power */
 		if (reg_write_byte(client, EPC660_REG_POWER_CTRL, 0x07) < 0)
 			return -EIO;
-		if (reg_write_byte(client, EPC660_REG_LED_DRIVER, 0xe0) < 0)
-			return -EIO;
+		// Do not set the LED driver register as the application shall have the possibilty to do that by hand
+		// 	if (reg_write_byte(client, EPC660_REG_LED_DRIVER, 0xe0) < 0)
+		//		return -EIO;
 		/* Switch to multi frame mode and enable shutter */
 		//		do not switch on the automatic shutter as the application shall have the possibilty to do that by hand
 		//		if (reg_write_byte(client, EPC660_REG_SHUTTER_CTRL, 0x03) < 0)
-- 
2.46.0

